autopeering/peer/peerdb.go

```diff
@@ -181,7 +181,11 @@ func (db *DB) LocalPrivateKey() (privateKey ed25519.PrivateKey, err error) {
 
 // UpdateLocalPrivateKey stores the provided key in the database.
 func (db *DB) UpdateLocalPrivateKey(key ed25519.PrivateKey) error {
-	return db.store.Set(localFieldKey(dbLocalKey), key.Bytes())
+	err := db.store.Set(localFieldKey(dbLocalKey), key.Bytes())
+	if err != nil {
```

Comment:
Hey!
Thanks for the issue report and the fix!
Can you change the following? :)

```suggestion
	if err := db.store.Set(localFieldKey(dbLocalKey), key.Bytes()); err != nil {
```

Reply:
Done